docs/platform/webcontainers/browser-config.md

```diff
@@ -85,7 +117,7 @@ Microsoft Edge disables usage of [WebAssembly][MDN_WEB_ASSEMBLY] when you set it
 
 To allow StackBlitz projects to run, either set this option to “Balanced” or add exceptions for a couple domains used by StackBlitz projects.
 
-1. Go to [edge://settings/privacy](edge://settings/privacy) → “Enhance your security on the web”.
+1. Go to `edge://settings/privacy` → “Enhance your security on the web”.
```

Comment:
I haven't tested again in Edge, but given that it's based on Chromium I expect that the same linking restrictions apply.